### PR TITLE
Remove year from license

### PR DESCRIPTION
**Pull request by Arman-Hosseini**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2006-2018 Doctrine Project
+Copyright (c) 2006-2019 Doctrine Project
```
Comment (Ocramius) [inline LICENSE]: Probably better to just drop the year overall

Comment (Ocramius) [inline LICENSE]: Probably better to just drop the year overall

**Review by Ocramius — DISMISSED** [review]

File: LICENSE (hunk 1 above)
Comment (Arman-Hosseini) [reply LICENSE]: Some people think that 2006 was the beginning of the project.
It's good to always be mentioned this year as the end.

Comment (Arman-Hosseini) [reply LICENSE]: Do you mean to remove the end of this year in general?

Comment (Ocramius) [reply LICENSE]: I'd remove the whole range - it is something that can be always extracted via our signed releases and commits.

Comment (greg0ire) [reply LICENSE]: Unresolving because the range is still here… did you forget to push your changes, maybe?

Comment (Arman-Hosseini) [reply LICENSE]: I did not get you.
@greg0ire

Comment (greg0ire) [reply LICENSE]: That conversation was marked as resolved, but the issue is still here. You may mark it as resolved once this is fixed, like so:
```suggestion
Copyright (c) Doctrine Project
```

Comment (Arman-Hosseini) [reply LICENSE]: Oh, yes, my changes have not been saved.
I added the changes again.
@greg0ire  Thank you.

**Review by lcobucci — APPROVED** [review]
@Arman-Hosseini awesome, thanks!

**Pull request merged**